src/buffer/out/AttrRow.hpp

```diff
@@ -28,7 +29,13 @@ class ATTR_ROW final
 public:
     using const_iterator = typename AttrRowIterator;
 
-    ATTR_ROW(const UINT cchRowWidth, const TextAttribute attr);
+    ATTR_ROW(const UINT cchRowWidth, const TextAttribute attr)
+    noexcept;
+
+    ATTR_ROW(ATTR_ROW&) = delete;
+    ATTR_ROW& operator=(ATTR_ROW&) = delete;
+    ATTR_ROW(ATTR_ROW&&) = default;
+    ATTR_ROW& operator=(ATTR_ROW&&) = default;

```

Comment:
funny enough, I was just working on some code that requires me to copy an ATTR_ROW. Is this off the table with small_vec?

Reply:
I don't think so - I just wanted to avoid unnecessary copies so making it not possible seemed nice since nobody needed it.  If you do, adding it should be ok.  small_vec will still use the move ctor when it can and having the constructor be noexcept allows it to offer the strong guarantee.